### ISSUE TITLE
Inconsistent with parentUID description in help file: only one part of the aircraft model can have no parentUID, but in the example file both the fuselage and wing have no parentUID defined.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ebouhoubeiny/cpacs#320]
```
Occurs in xPath: 
/cpacs/vehicles/aircraft/model/wings/wing

Comments: 
Inconsistent with parentUID description in help file: only one part of the 
aircraft model can have no parentUID, but in the example file both the fuselage 
and wing have no parentUID defined.

Reply: 
there needs to be at least one part without a parentUID, also take care of this 
with CPACS 1.7 releasebird

daniel.boehnke@dlr.de
```

Original issue reported on code.google.com by `daniel.b...@googlemail.com` on 25 Jan 2012 at 10:57

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 8 Feb 2012 at 9:46

**Comment by GoogleCodeExporter** [comment]
```
fixed in CPACS 2.0 and 1.7 releaseBirds
```

Original comment by `daniel.b...@googlemail.com` on 9 Feb 2012 at 10:55
- Changed state: **Fixed**
